Invert block-webaudio default (#24314)
mdn/content: files/en-us/web/media/autoplay_guide/index.md

- `media.autoplay.allow-muted`
  - : A Boolean preference which if `true` (the default) allows audio media which is currently muted to be automatically played. If this has been changed to `false`, media with an audio track will not be permitted to play even if muted.
- `media.autoplay.block-webaudio`

>   - : A Boolean preference that indicates whether to apply autoplay blocking to the [Web Audio API](/en-US/docs/Web/API/Web_Audio_API).
>      If false, web audio is always allowed to autoplay.
>      When true, audio contexts are only able to play on pages once there has been {{Glossary("Sticky activation")}}.
>      The default is set to `true`.

Reviewer: ```suggestion
  - : A Boolean preference that indicates whether to apply autoplay blocking to the [Web Audio API](/en-US/docs/Web/API/Web_Audio_API).
     If `false`, web audio is always allowed to autoplay.
     If `true`, audio contexts are only able to play on pages once there has been {{Glossary("Sticky activation")}}.
     The default is set to `true`.
```

- `media.autoplay.allow-muted`
  - : A Boolean preference which if `true` (the default) allows audio media which is currently muted to be automatically played. If this has been changed to `false`, media with an audio track will not be permitted to play even if muted.
- `media.autoplay.block-webaudio`

>   - : A Boolean preference which indicates whether to apply autoplay blocking to the [Web Audio API](/en-US/docs/Web/API/Web_Audio_API). The default is set to `true`, in which the autoplay of the sound is disabled.

Reviewer: ```suggestion
  - : A Boolean preference that indicates whether to apply autoplay blocking to the [Web Audio API](/en-US/docs/Web/API/Web_Audio_API).
     The default is set to `true` (autoplay of media with sound is disabled).
```

- `media.autoplay.allow-muted`
  - : A Boolean preference which if `true` (the default) allows audio media which is currently muted to be automatically played. If this has been changed to `false`, media with an audio track will not be permitted to play even if muted.
- `media.autoplay.block-webaudio`

>   - : A Boolean preference that indicates whether to apply autoplay blocking to the [Web Audio API](/en-US/docs/Web/API/Web_Audio_API).
>      The default is set to `true`.

Reviewer: I think we need to be clear what allows autoplay in this case - waiting on response in https://bugzilla.mozilla.org/show_bug.cgi?id=1773577#c6

```suggestion
  - : A Boolean preference that indicates whether to apply autoplay blocking to the [Web Audio API](/en-US/docs/Web/API/Web_Audio_API).
     When this is set, audio contexts are only able to play on pages once there has been {{Glossary("Sticky activation")}}.
     The default is set to `true`.
```

>      The default is set to `true` (autoplay of media with sound is disabled).

Reviewer: Actually, that "autoplay of media with sound is disabled is incorrect/incomplete". 
```suggestion
     The default is set to `true`.
```
